exampleCourse/questions/element/hiddenHints/server.py

```diff
@@ -0,0 +1,8 @@
+# Sets the condition for the special hint to be shown based on the submitted
+# value. Note that the hint is only shown if the student hits "Save & Grade"
+# with a valid submission. It is also possible to set this in `parse(data)`,
+# which would cause the hint to be shown in case of a "Save only" submission or
+# if the submission is invalid (e.g., blank or with non-numeric values).
+def grade(data):
+    if "c_2" in data["submitted_answers"] and data["submitted_answers"]["c_2"] == 10:
```

Comment:
Won't block on this, but you can retrieve the "c_2" key using a .get call and avoid the first check.

Reply:
That thread specifically suggested using `.get()`, let's do that for concision.